custom/icds_reports/utils/aggregation_helpers/distributed/agg_child_health.py

```diff
@@ -192,14 +192,7 @@ def staging_queries(self):
 
     def update_queries(self):
         yield f"""
-            CREATE TABLE "local_tmp_agg_child_health" AS SELECT * FROM "{self.temporary_tablename}";
-            INSERT INTO "{self.staging_tablename}" SELECT * from "local_tmp_agg_child_health";
-            DROP TABLE "local_tmp_agg_child_health";
-        """, {
-        }
-
-        yield f"""
-        UPDATE "{self.staging_tablename}" agg SET
+            UPDATE "{self.temporary_tablename}" agg SET
```

Comment:
could potentially add this to the initial creation query, but this does seem better than the current workflow